waffle-mock-contract/src/index.ts

```diff
@@ -32,7 +32,8 @@ function createMock(abi: ABI, mockContractInstance: Contract) {
 
   return Object.values(functions).reduce((acc, func) => ({
     ...acc,
-    [func.name]: stub(mockContractInstance, encoder, func)
+    [func.name]: stub(mockContractInstance, encoder, func),
+    [func.format()]: stub(mockContractInstance, encoder, func)
```

Comment:
the stub could only be created once and assigned twice, i.e. reuse `stub(mockContractInstance, encoder, func)`